packages/dev/s2-docs/pages/s2/styling/reference.mdx

```diff
@@ -0,0 +1,80 @@
+import {Layout} from '../../../src/Layout';
+import {InlineAlert, Heading, Content, Link} from '@react-spectrum/s2';
+import {S2Colors} from '../../../src/S2Colors';
+import {S2Typography} from '../../../src/S2Typography';
+import {StyleMacroProperties} from '../../../src/types';
+import {getPropertyDefinitions, getShorthandDefinitions} from '../../../src/styleProperties';
+export default Layout;
+
+export const omitFromNav = true;
+export const tags = ['style', 'macro', 'spectrum', 'custom', 'values', 'reference'];
+export const description = 'Reference table for the `style` macro';
+
+# Style Macro Reference
+
+Reference for the properties and values supported by React Spectrum `style` macro.
+
+## Colors
+
+All Spectrum 2 color tokens are available across color properties (e.g., `backgroundColor`, `color`, `borderColor`).
+
+<S2Colors />
+
+<StyleMacroProperties properties={getPropertyDefinitions('color')} />
```

Comment:
Nit, we should add some margin between these:

<img width="313" height="80" alt="Image" src="https://github.com/user-attachments/assets/f7559270-4e56-4398-98a1-6e817daabb91" />